docs/sources/write/shortcodes/index.md

```diff
@@ -333,6 +333,121 @@ compactor:
 
 {{< /collapse >}}
 
+## Guide
+
+The `guide` shortcode embeds a Grot guide into the docs that people can follow to get help and guidance on topics and direction to documentation resources.
+
+```markdown
+{{</* guide name="instrument" title="title" text="text" button="button" */>}}
+```
+
+The website looks up guide Markdown files from the website repository in the `content/guides` folder. The shortcode accepts the following parameters:
+
+| Parameter | Description                                                                 | Required |
+| --------- | --------------------------------------------------------------------------- | -------- |
+| `name`    | The name of the guide Markdown file in the website `content/guides` folder. | yes      |
+| `title`   | A custom guide title to override the default guide title.                   | no       |
+| `text`    | Custom body text to override the default guide text.                        | no       |
+| `button`  | Custom button copy to override the default guide button copy.               | no       |
+
+If you don't set custom title, text, and button values, the guide introduction screen will use the following default text:
+
+![Grot guide introduction screen with default copy](https://grafana.com/media/docs/writers-toolkit/grot-guide.png)
```

Comment:
For accessibility, should we replace the image with the actual text for each parameter?

Reply:
It's also to give a visual representation of the grot guide.